### PR TITLE
Indicate "read" vs "unread" conversations

### PR DESCRIPTION
**Pull request by reichert621**
unread-vs-read

### CODE REVIEW

File: assets/src/components/demo/Demo.tsx [hunk 1]
```diff
@@ -99,6 +99,7 @@ class Demo extends React.Component<Props, State> {
           subtitle={subtitle}
           primaryColor={color}
           accountId={accountId}
+          baseUrl="http://localhost:4000"
```
Comment (cheeseblubber) [inline assets/src/components/demo/Demo.tsx:102]: Lets make this app.papercups.io by default

Comment (reichert621) [reply assets/src/components/demo/Demo.tsx:102]: oops forgot to revert, good catch!

File: lib/chat_api/slack.ex [hunk 2]
```diff
@@ -110,6 +102,34 @@ defmodule ChatApi.Slack do
     end
   end
 
+  def get_initial_slack_thread_subject(conversation_id) do
+    # TODO: use env variables here?
```
Comment (cheeseblubber) [inline lib/chat_api/slack.ex:106]: Made a ticket for all todos https://www.pivotaltracker.com/story/show/173993344

Comment (cheeseblubber) [inline lib/chat_api/slack.ex:106]: Made a ticket for all todos https://www.pivotaltracker.com/story/show/173993344

**Pull request merged**